rest/diagnostic_doc_api_test.go

```diff
@@ -57,3 +61,70 @@ func TestGetAlldocChannels(t *testing.T) {
 	assert.Equal(t, len(channelMap["CHAN3"]), db.DocumentHistoryMaxEntriesPerChannel+1)
 
 }
+
+func TestGetDocDryRuns(t *testing.T) {
```

Comment:
This looks like it needs negative tests for the error cases (no body, no docid, etc)